### PR TITLE
Made some improvements to hello world example based on user feedback

### PR DESCRIPTION
**Pull request by gunamata**
Signed-off-by: Gunasekhar Matamalam <gunasekhar.matamalam@suse.com>

### CODE REVIEW

File: docs/how-to-guides/hello-world-example.md [hunk 1]
```diff
@@ -65,73 +65,58 @@ FROM nginx:alpine
...
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl stop <container-id>
-nerdctl rm <container-id>
-nerdctl ps #to verify nothing is running 
-nerdctl rmi nginx-helloworld:v1.0 #to remove the image
+kubectl delete pod hello-world
+nerdctl rmi nginx-helloworld:latest --namespace k8s.io #to remove the image
```
Comment (jandubois) [inline docs/how-to-guides/hello-world-example.md:110]: The `--namespace` option must come **before** `rmi`:

```suggestion
# remove the image
nerdctl --namespace k8s.io rmi nginx-helloworld:latest
```

File: docs/how-to-guides/hello-world-example.md [hunk 2]
```diff
@@ -65,73 +65,58 @@ FROM nginx:alpine
...
   </TabItem>
   <TabItem value="docker">
 
 ```
-docker stop <container-id>
-docker rm <container-id>
-docker ps #to verify nothing is running 
-docker rmi nginx-helloworld:v1.0 #to remove the image
+kubectl delete pod hello-world 
+docker rmi nginx-helloworld:latest #to remove the image
```
Comment (jandubois) [inline docs/how-to-guides/hello-world-example.md:118]: I would prefer comments on a separate line and not hidden at the end of a command

```suggestion
# remove the image
docker rmi nginx-helloworld:latest
```

**Review by jandubois — CHANGES_REQUESTED** [review]

File: docs/how-to-guides/hello-world-example.md [hunk 3]
```diff
@@ -65,73 +65,58 @@ FROM nginx:alpine
...
+:warning: **Note:** Please note that you need to pass the flag `--namespace k8s.io` to the `nerdctl` build command, so that `nerdctl` builds the image and then makes it available in the `k8s.io` namespace.
 
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl build -t nginx-helloworld:v1.0 .
-nerdctl images | grep nginx-helloworld
-nerdctl run -d -p 8080:80 --name my-site nginx-helloworld:v1.0
+nerdctl build --namespace k8s.io -t nginx-helloworld:latest .
```
[the remark this answers is not in the document]
Comment (gunamata) [reply docs/how-to-guides/hello-world-example.md]: Fixed the command per your suggestion. Also, changed to long options at other places in this doc.

File: docs/how-to-guides/hello-world-example.md [hunk 4]
```diff
@@ -65,73 +65,58 @@ FROM nginx:alpine
 COPY . /usr/share/nginx/html
 ```
 
-**Build and run the image for verification purposes:**
+**Build Image from code locally:**
```
[the remark this answers is not in the document]
Comment (gunamata) [reply docs/how-to-guides/hello-world-example.md]: Changed to Level 4 heading in this doc. I agree we need to do this in other docs too, with an another PR.

File: docs/how-to-guides/hello-world-example.md (hunk 1 above)
Comment (gunamata) [reply docs/how-to-guides/hello-world-example.md]: Modified per your suggestion.

File: docs/how-to-guides/hello-world-example.md (hunk 2 above)
Comment (gunamata) [reply docs/how-to-guides/hello-world-example.md]: Moved comments to their own lines.

File: docs/how-to-guides/hello-world-example.md [hunk 5]
```diff
@@ -7,131 +7,120 @@ import TabItem from '@theme/TabItem';
...
 
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl build -t nginx-helloworld:v1.0 .
-nerdctl images | grep nginx-helloworld
-nerdctl run -d -p 8080:80 --name my-site nginx-helloworld:v1.0
+nerdctl --namespace k8s.io build --tag nginx-helloworld:latest .
+nerdctl images --namespace k8s.io | grep nginx-helloworld
```
Comment (jandubois) [inline docs/how-to-guides/hello-world-example.md:79]:
```suggestion
nerdctl --namespace k8s.io images | grep nginx-helloworld
```

File: docs/how-to-guides/hello-world-example.md [hunk 6]
```diff
@@ -7,131 +7,120 @@ import TabItem from '@theme/TabItem';
...
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl build -t helloworld:v1.0 .
+nerdctl build --tag helloworld:v1.0 .
 nerdctl images | grep helloworld
 nerdctl run --rm helloworld:v1.0
-nerdctl rmi helloworld:v1.0 #To remove the image
+#To remove the image
```
Comment (jandubois) [inline docs/how-to-guides/hello-world-example.md:35]: Nitpicking, but I think there should always be a space between `#` and the first character.
Also `To ` is just meaningless filler.
```suggestion
# Remove the image
```

Comment (gunamata) [reply docs/how-to-guides/hello-world-example.md:35]: Modified per your suggestion.

File: docs/how-to-guides/hello-world-example.md (hunk 5 above)
Comment (gunamata) [reply docs/how-to-guides/hello-world-example.md]: I missed to fix to this in my earlier commit. Anyways, fixed it now.

File: docs/how-to-guides/hello-world-example.md (hunk 6 above)
Comment (jandubois) [reply docs/how-to-guides/hello-world-example.md:35]: > Modified per your suggestion.

But not everywhere, and not here either. :)

Comment (gunamata) [reply docs/how-to-guides/hello-world-example.md]: Done :)

**Review by jandubois — APPROVED** [review]

**Pull request merged**